examples/tutorials/ordinary_model_example.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>No need for <code>before</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>